pkg/minikube/cluster/commands.go

```diff
@@ -91,7 +91,7 @@ func GetStartCommandSystemd(kubernetesConfig KubernetesConfig, localkubeStartCmd
 	if err := t.Execute(&buf, data); err != nil {
 		return "", err
 	}
-    Mkdir("/usr/lib/systemd/system/",FileMode(0755))
+    os.Mkdir("/usr/lib/systemd/system/",os.FileMode(0755))
```

Comment:
Can you move this folder creation to https://github.com/kubernetes/minikube/blob/master/pkg/minikube/machine/drivers/none/none.go#L158
It makes more sense to be part of the none driver start function.